08-projects.Rmd

```diff
@@ -5155,9 +5190,9 @@ SELECT * FROM users AS u LEFT JOIN dorms AS d ON u.dorm=d.id;
 
 For example:
 
-`(84) Ryland Park @ (37.342725,-121.895617)`
+`(84) Ryland Park @ (37.342725, -121.895617)`
 
-**Relevant topics:** [aliasing](#sql-examples), [concat](#cat)
+**Relevant topics:** [aliasing](#sql-aliasing), concat
```

Comment:
Hi @kevinamstutz , for `concat`, I removed the link as it directs to a subsection under Unix. As you already provided a link for `||`, perhaps we could skip the link for `concat` here.

Reply:
I agree completely.